Source/AppAuthCore/OIDTokenRequest.m

```diff
@@ -330,7 +330,7 @@ - (NSURLRequest *)URLRequest {
   }
   
   for (id header in _additionalHeaders) {
-    [URLRequest setValue:httpHeaders[header] forHTTPHeaderField:header];
+    [URLRequest setValue:_additionalHeaders[header] forHTTPHeaderField:header];
```

Comment:
Good catch! Would you be able to add a unit test verifying that `-[OIDTokenRequest URLRequest]` has the correct additional parameters?

Reply:
I wished I could add, but Objective C isn't my native language.

Reply:
@mdmathias 
I've tried to add test. It works, but I'm not sure that everything has been written correctly and in a right place.